tests/integration/records/delete-record-test.js

```diff
@@ -1,4 +1,4 @@
-/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "(adam|dave|cersei)" }]*/
+3/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "(adam|dave|cersei)" }]*/
```

Comment:
I had actually fixed this and "pushed", but I forgot to actually `git add` the file so it was in the commit… Thanks @sduquej :)